flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -249,6 +249,7 @@ int32_t transmitter_control_update()
 
 		if (settings.ChannelGroups[n] >= MANUALCONTROLSETTINGS_CHANNELGROUPS_NONE) {
 			cmd.Channel[n] = PIOS_RCVR_INVALID;
+			validChannel[n] = false;
```

Comment:
Ah, right.  Scaled version is not an input to ```validInputRange()```.  It's still sketchy to be passing an error code into those functions at all.  The ```if``` on (new) line 260 only tests against ```PIOS_RCVR_TIMEOUT``` and misses the other 2 potential error values.